resources/assets/lib/components/comment.coffee

```diff
@@ -198,6 +210,26 @@ export class Comment extends React.PureComponent
               label: osu.trans('comments.load_replies') if @children.length == 0
               ref: @loadMoreRef
 
+  
+  renderBlocked: =>
```

Comment:
should take `mainClass` and `blockClass` param so the values from `render` can just be passed in.